src/sdam/topology.ts

```diff
@@ -903,10 +903,22 @@ function processWaitQueue(topology: Topology) {
       continue;
     } else if (selectedDescriptions.length === 1) {
       selectedServer = topology.s.servers.get(selectedDescriptions[0].address);
+    } else if (selectedDescriptions.length === 2) {
+      // If there are only two servers, we avoid shuffling the array of
+      // server descriptions
+      const server1 = topology.s.servers.get(selectedDescriptions[0].address);
+      const server2 = topology.s.servers.get(selectedDescriptions[1].address);
+
+      if (server1?.s.operationCount === server2?.s.operationCount) {
+        selectedServer = Math.floor(Math.random() * 2) === 0 ? server1 : server2;
```

Comment:
I am assuming this is for performance to avoid using the `shuffle` util method?

Reply:
Yup!